Packaging/amiga/prep.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/bash
+
+# exit when any command fails
+set -e
```

Comment:
`set -euo pipefail` is the right spell here

`u` errors on use of undefined variables
`o pipefail` errors when an operation left of a `|` fail (`-e` alone doesn't do this)